resources/js/app.js

```diff
@@ -7,12 +8,14 @@ import { InertiaForm } from 'laravel-jetstream';
 import PortalVue from 'portal-vue';
 import Calendar from 'v-calendar/lib/components/calendar.umd'
 import DatePicker from 'v-calendar/lib/components/date-picker.umd'
+import DateFormatter from "../js/Plugins/date-formatter";
```

Comment:
```suggestion
import DateFormatter from "@src/Plugins/date-formatter";
```

Use the aliases for cleaner imports!

Reply:
Ya I initially tried multiple ways of importing which caused errors in the CI, this is the first one that worked and forgot to clean it up. Good catch